src/User32.Desktop/User32.cs

```diff
@@ -647,13 +1103,71 @@ public static partial class User32
         [DllImport(nameof(User32), SetLastError = true, CharSet = CharSet.Unicode)]
         public static extern int RegisterWindowMessage(string lpString);
 
+        [DllImport(nameof(User32), SetLastError = true)]
+        public static extern bool GetWindowDisplayAffinity(IntPtr hWnd, out int dwAffinity);
+
+        [DllImport(nameof(User32), SetLastError = true)]
+        public static extern bool EnumDisplayMonitors(IntPtr hdc, IntPtr lprcClip, MonitorEnumDelegate lpfnEnum, IntPtr dwData);
+
+        [DllImport(nameof(User32), SetLastError = true, CharSet = CharSet.Unicode)]
+        public static extern bool GetClassInfoEx(IntPtr hInstance, string lpClassName, ref WNDCLASSEX lpWndClass);
+
+        [DllImport(nameof(User32), EntryPoint = "GetMonitorInfo", SetLastError = true)]
+        public static extern unsafe bool GetMonitorInfo(
+            IntPtr hMonitor,
+            [Friendly(FriendlyFlags.Out | FriendlyFlags.Optional)] MONITORINFO* lpmi);
+
+        [DllImport(nameof(User32), EntryPoint = "GetMonitorInfo", SetLastError = true)]
```

Comment:
Ah, I think you're right about the generated overloads colliding with each other. Good point. Rats. I wonder if we should omit the `GetMonitorInfoEx` method altogether until we can find a fix that allows our methods to keep the names of the native ones. If you are OK with this, can you keep your overload, but just double commented (`////`) so that we see it and can fix it up later? (perhaps open an issue to track)